src/gamekit.cs

```diff
@@ -1603,11 +1604,13 @@ interface GKAchievement : NSSecureCoding {
 
 
 #if MONOMAC
```

Comment:
It's not possible to remove this '#if MONOMAC`?

Reply:
Fixed.